scripts/build/Jenkins/Jenkinsfile

```diff
@@ -576,7 +568,7 @@ def CreateSingleNode(Map pipelineConfig, def platform, def build_job, Map envVar
                                 error "Node disconnected during build: ${e}"  // Error raised to retry stage on a new node
                             }
                         }
-                        if (IsAPLogUpload(branchName, build_job_name)) {
+                        if (jobName.toLowerCase().contains('asset') && env.IS_UPLOAD_AP_LOGS) {
```

Comment:
Is `env.IS_UPLOAD_AP_LOGS` a Jenkins env var? Just making sure that's something we set before we deploy.

Reply:
Yeah, it's a global Jenkins env var, we will set it on our fork Jenkins first, and we can later set it on O3DE Jenkins if we need to archive AP logs for O3DE.

Reply:
Does this also mean we're changing the criteria to upload AP logs. Looks like it will now upload for all asset jobs if we have the global env var set.

Reply:
Yeah, we will upload AP logs for all asset jobs once this is enabled, before we didn't upload logs for PR builds, but there were requests asking for AP logs from PR builds so I just enabled it.

There is a config to delete the artifacts when the build is deleted, so those logs will be deleted once the branches or PRs get deleted.